static/img/c2c/taxii/example.tf

```diff
@@ -0,0 +1,19 @@
+resource "sumologic_cloud_to_cloud_source" "taxii_source" {
```

Comment:
As this is a .tf file, it should be moved to the `static/files` folder (currently it's in the images folder).

Reply:
You are correct Kim, it should be under static/files. But when we did the C2C cleanup, we moved all these examples to images folder by mistake. 
I will take this as a separate task and will move all the existing .json and .tf examples from img folder to  files folder.

Reply:
That sounds good, thanks for handling! Here's the ticket i've made. and i'll approve this PR https://app.asana.com/0/1183741177809740/1206145385179444/f